docs/docs/recipes.md

```diff
@@ -130,7 +130,65 @@ Data sourcing in Gatsby is plugin-driven; Source plugins fetch data from their s
 
 ## Transforming data
 
-Transforming data in Gatsby is also plugin-driven; Transformer plugins take data fetched using source plugins, and process it into something more usable (e.g. JSON into JavaScript objects, markdown to HTML, and more).
+Transforming data in Gatsby is also plugin-driven. Transformer plugins take data fetched using source plugins, and process it into something more usable (e.g. JSON into JavaScript objects, markdown to HTML, and more). One such Transformer plugin is gatsby-transformer-plugin that you can use to transform your markdown files to HTML.
 
-- Walk through an example using the `gatsby-transformer-remark` plugin to transform markdown files [tutorial part six](/tutorial/part-six/#transformer-plugins)
-- Search available transformer plugins in the [Gatsby library](/plugins/?=transformer)
+### Requirements
+
+- A gatsby site with `gatsby-config.js` and `index.js` page
+- A Markdown file saved in your Gatsby site `src` directory
+- The `gatsby-transformer-remark` plugin using `npm install --save gatsby-transformer-remark` command
```

Comment:
```suggestion
- The `gatsby-transformer-remark` plugin installed
```

Since recipes are shorter and don't need to include setup instructions I think it'd be fine to just say that the plugin is installed

Reply:
Agreed! I was a bit doubtful about this instruction myself.